src/components/pages/EntitySearch.vue

```diff
@@ -373,19 +373,12 @@ export default {
     },
 
     getMatchDetails(entity) {
-      const target = entity.matched_terms?.[0]?.[0]
-      const term = entity.matched_terms?.[0]?.[1]
-      if (target?.startsWith('data_')) {
-        return this.$t('search.match_details_2', {
-          term,
-          target: target.replace('data_', '')
-        })
-      } else {
-        return this.$t('search.match_details', {
-          term,
-          target
-        })
-      }
+      const target = entity.matched_terms.join(', ')
+      const term = this.searchQuery
+      return this.$t('search.match_details', {
+        term,
```

Comment:
`term` it no longer exists in that updated translation: `match_details: 'match found in {target}'`

Reply:
Thx!